pandlas/SqlRace.py

```diff
@@ -325,7 +329,7 @@ def __init__(
         if self.mode == "r":
             self.load_session(session_key)
         elif self.mode == "w":
-            self.create_sqlrace()
+            self.create_sqlrace(ip_address)
```

Comment:
not needed as self.ip_address is set in L321?

Reply:
removed ip_address from the self.create_sqlrace()